README.md

```diff
@@ -5,167 +5,144 @@
+Ark consists of:
 
-More concretely, Heptio Ark combines an in-cluster service with a CLI that allows you to record both:
-1. *Configurable subsets of Kubernetes API objects* -- as tarballs stored in object storage
-2. *Disk snapshots of Persistent Volumes* -- via the cloud provider APIs
+* A server that runs on your cluster
+* A CLI
 
-Heptio Ark currently supports the [AWS][15], [GCP][16], and [Azure][17] cloud provider platforms.
+## Getting started
 
-## Quickstart
+The following example sets up the Ark server and client, then backs up and restores a sample application.
 
-This guide gets Ark up and running on your cluster, and goes through an example using the following:
-* **Minio, an S3-compatible storage service** that runs locally on your cluster. This is the storage service where backup files are uploaded. *Note that Ark is intended to run on a cloud provider--we are using Minio here to keep the example convenient and self-contained.*
+For simplicity, the example uses Minio, an S3-compatible storage service that runs locally on your cluster. See [Set up Ark with your cloud provider][3] for how to run on a cloud provider. 
 
-* **A sample nginx app** under the `nginx-example` namespace, used to demonstrate Ark's backup and restore functionality.
+### Prerequisites
 
-Note that this example *does not* include a demonstration of PV disk snapshots, because that feature requires integration with a cloud provider API. For snapshotting examples and instructions specific to AWS, GCP, and Azure, see [Cloud Provider Specifics][23].
+* Access to a Kubernetes cluster, version 1.7 or later
+* A DNS server on the cluster
+* `kubectl` installed
 
-### 0. Prerequisites
+### Download
 
-* *You should have access to an up-and-running Kubernetes cluster (minimum version 1.7).* If you do not have a cluster, [choose a setup solution][9] from the official Kubernetes docs.
+Clone or fork the Ark repository:
 
-* *You will need to have a DNS server set up on your cluster for the example files to work.* You can check this with `kubectl get svc -l k8s-app=kube-dns --namespace=kube-system`. If said service does not exist, [these instructions][12] may help.
-
-* *You should have `kubectl` installed.* If not, follow the instructions for [installing via Homebrew (MacOS)][10] or [building the binary (Linux)][11].
-
-### 1. Download
-Clone or fork the Heptio Ark repo:
 ```
 git clone git@github.com:heptio/ark.git
 ```
 
-> NOTE: Documentation may change between releases. See the [Changelog][20] for links to previous versions of this repository and its docs.
->
-> To ensure that you are working off a specific release, `git checkout <VERSION_TAG>` where `<VERSION_TAG>` is the appropriate tag for the Ark version you wish to use (e.g. "v0.3.3"). You should `git checkout master` only if you're planning on [building the Ark image from scratch][7].
+NOTE: Make sure to check out the appropriate version.
```

Comment:
Is it worth expanding just a little bit here, suggesting they check out the latest git tag, and commenting that master is under active development and things may be in flux?